index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>
+        <header class="navbar">
+        <div class="container">
+            <div class="logo"><a href="#"><img src="img/My Logo.png"></a></div>
+        <div class="menu"><img src="img/Union.svg" alt=""></div>
```

Comment:
[OPTIONAL] I suggest you have this inside a `<nav>` element. You will need this when you activate it using javascript to access nav links